doc/releases/release-notes-3.6.rst

```diff
@@ -577,6 +577,10 @@ Networking
 
 * LwM2M:
 
+  * Added new API function to set multiple resource values:
```

Comment:
I'm afraid this won't make it to 3.6, will need to move this entry to 3.7 release notes, once available.

Reply:
OK. Removed the change note entry for now.